src/components/image-url-input.js

```diff
@@ -0,0 +1,75 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import styled from 'styled-components/macro';
+import { Flex } from 'rebass';
+
+import { Input } from './';
+
+const ImageUrlInput = ({ visible, value, onChange, transitionName }) => {
+  const TransitionContainer = useMemo(
+    () =>
+      styled(Flex)`
+        &.${transitionName}-enter {
+          opacity: 0.01;
+          max-height: 0;
```

Comment:
Is there a reason to animate `max-height` instead of `height`?

Reply:
yeah, because we don't want to set the height explicitly, and the animation wouldn't work if we used percentage value (aka `height: 100%`)